src/components/Web3ConnectButton.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react'
+import Web3Connect from 'web3connect'
+// API
+import getWeb3Api from '../api/web3'
+// Components
+import DisplayWeb3 from './DisplayWeb3'
+// Types
+import { provider as ProviderType, Web3Api } from '../types'
+
+const Web3ConnectButton: React.FC = () => {
+    const [web3Api, setWeb3FrontendAPI] = useState<Web3Api>()
```

Comment:
I think `Web3FrontendApi` was renamed to `Web3Api`. 

setWeb3FrontendAPI should also be coherent